scripts/run_test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+docker system prune --all --force
```

Comment:
This is a pretty forceful command that can break things for people. I think you should just add a `--no-cache` flag to the docker commands used.

Reply:
Indeed, I designed this script to be run on a machine for CI

Reply:
Sure, but someone might want to run this for personal testing as well.